### PR TITLE
[WIP] : Update stale dependencies

### PR DESCRIPTION
**Pull request by sharathsrinivasa**

### CODE REVIEW

**Comment by fabiocarvalho777** [comment]
@sharathsrinivasa let me know when this is ready for review. Thanks.

File: build.gradle [hunk 1]
```diff
@@ -45,6 +45,7 @@ subprojects {
 //            outputs.upToDateWhen {false}
 //            showStandardStreams = true
         }
+        exclude '**/PerformanceIT.class'
```
Comment (sharathsrinivasa) [inline build.gradle]: TODO : Remove this exclusion, this was only done to test in local.

**Comment by sharathsrinivasa** [comment]
@fabiocarvalho777 , is there a CI job setup for PR?

**Comment by fabiocarvalho777** [comment]
> @fabiocarvalho777 , is there a CI job setup for PR?

Yes, TavisCI. For some reason it seems to not be working. Let me take a look.

**Comment by sharathsrinivasa** [comment]
> > @fabiocarvalho777 , is there a CI job setup for PR?
> 
> Yes, TavisCI. For some reason it seems to not be working. Let me take a look.

were you able to fix the CI?

**Comment by fabiocarvalho777** [comment]
> > > @fabiocarvalho777 , is there a CI job setup for PR?
> > 
> > 
> > Yes, TavisCI. For some reason it seems to not be working. Let me take a look.
> 
> were you able to fix the CI?

Yes. I think you will only be able to see it though when you create a new PR.

**Comment by fabiocarvalho777** [comment]
@sharathsrinivasa let me know when this PR is ready for review

**Comment by sharathsrinivasa** [comment]
Closing this PR for now, will reopen soon.

**Pull request closed without merging**